content/header-footer.html

```diff
@@ -6,59 +6,70 @@
 <link rel="stylesheet" href="styles/main.css">
 <link rel="stylesheet" href="styles/styleguide.css">
 
-<section class="st-section">
-  <h1 class=st-title>Header</h1>
-
-  <h2 class=st-title>Not logged</h2>
-
-  <div class="st-container">
-    <header class=main-header>
-      <h1><a href="/">Neomad</a></h1>
-
-      <nav>
-        <a href={{ url_for('article_list') }} title="Read nomad's adventures">Read</a>
-        <a href={{ url_for('login') }} class=login>Log in</a>
-      </nav>
-    </header>
-  </div>
+<header class=main-header>
+  <h1><a href="index.html">Neomad - Styleguide</a></h1>
+  <nav>
+    <a href="atoms.html">Atoms</a>
+    <a href="molecules.html">Molecules</a>
+    <a href="header-footer.html">Header-Footer</a>
+  </nav>
+</header>
+
+<main>
+  <section class="st-section">
+    <h1 class=st-title>Header</h1>
+
+    <h2 class=st-title>Not logged</h2>
+
+    <div class="st-container">
+      <header class=main-header>
+        <h1>
+          <a href>Neomad</a>
```

Comment:
`href=/` or `href=https://neomad.org` 🙃

Reply:
we should stay in the styleguide repository